src/magellan_motion/src/path_planner/planner.cpp

```diff
@@ -86,8 +90,10 @@ Path PathPlanner::plan(Point goal) {
     std::chrono::time_point<std::chrono::high_resolution_clock> startTime =
         std::chrono::high_resolution_clock::now();
 
-    goalX = std::roundf(goal.x);
-    goalY = std::roundf(goal.y);
+    //casts the float value (precision to hundredth's place) to an int to be used by the planner
+    //this is so that we can handle floating values better (ex 1.97 --> 197
+    goalX = static_cast<int>(goal.x/0.01);
```

Comment:
I wanted to have this be goal.x/floatPrecisionDivide but I kept getting a warning about dividing by 0 so I left it as a hard-coded value.

Reply:
Why couldn't you just multiply by `floatPrecision`?

Reply:
Also, you need to transform the X and Y goal points into the base_link frame from their source frame before you cast and use them.